application-insights.tf

```
@@ -0,0 +1,24 @@
+module "application_insights" {
+  source = "git@github.com:hmcts/terraform-module-application-insights?ref=4.x"
+
+  env     = var.env
+  product = var.product
+  name    = var.product
+
+  resource_group_name = azurerm_resource_group.shared_resource_group.name
+
+  common_tags = var.common_tags
+
+  daily_data_cap_in_gb = var.daily_data_cap_in_gb
+}
+
+moved {
+  from = azurerm_application_insights.appinsights
+  to   = module.application_insights.azurerm_application_insights.this
+}
```

```suggestion
moved {
  from = azurerm_application_insights.appinsights
  to   = module.application_insights.azurerm_application_insights.this
}
```
Can probably remove if building from scratch?